runtime/src/main/java/com/dylibso/chicory/runtime/ConstantEvaluators.java

```diff
@@ -51,26 +52,45 @@ public static Value computeConstantValue(Instance instance, List<Instruction> ex
                     }
                 case REF_FUNC:
                     {
-                        tos = Value.funcRef((int) instruction.operands()[0]);
+                        var idx = (int) instruction.operands()[0];
+                        instance.function(idx);
```

Comment:
`function()` returns null for negative values, so this will succeed. Should this fail for negative values?

Reply:
Addressed in this way: https://github.com/dylibso/chicory/pull/421/commits/a423f05ab0ac5dd55a193b4909e74cfc0c117ce6

Reply:
Nice, that looks better